src/downloader.rs

```diff
@@ -22,7 +23,7 @@ pub enum Error {
     },
     #[snafu(display("The downloaded archive is empty"))]
     TarIsEmpty,
-    #[snafu(display("Can't find version upstream"))]
+    #[snafu(display("Can't find version/arch upstream"))]
```

Comment:
Can we print the arch here, because most users won't override it?

Reply:
Give a mouse a cookie...
<img width="705" alt="image" src="https://user-images.githubusercontent.com/2559043/112764902-33cd6380-8fd0-11eb-8c03-be1e5833a533.png">

Reply:
6638cb3